perly.y

```diff
@@ -112,6 +114,7 @@
 %type <pval>  fieldvar /* pval is PADNAME */
 %type <opval> optfieldattrlist fielddecl
 %type <opval> termbinop termunop anonymous termdo
+%type <svval> optopflags
```

Comment:
The new non-terminal needs clean-up in S_clear_yystack() in perly.c.

It could use dumping support in yy_stack_print() in perly.c too.